includes/Modules/Analytics_4/Audience_Settings.php

```diff
@@ -108,7 +121,7 @@ public function merge( $settings ) {
 	 *
 	 * @return array The sanitized settings.
 	 */
-	private function sanitize( $option ) {
+	private function sanitize( $option = array() ) {
```

Comment:
Why has this default value been added? It's not something that's typically needed for our sanitize callbacks...